safe-global/safe-wallet-web
# [Multichain] fix: Show notification when adding a new network to an account [SW-173]

## What it solves

Resolves [SW-173](https://www.notion.so/safe-global/Close-networks-selector-in-the-header-after-choosing-network-8300245b7058465ea6de4a9e9171f094)

## How this PR fixes it

- Shows a notification when replaying a safe on another network
- Closes the Network selector when closing the add new network form
- Adds tab navigation to open the network selector collapsed area
- Removes unused `onChange` handler and moves the track event call to the form submission

## How to test it

1. Open a Safe
2. Add new network via the header network selector
3. Observe the network selector disappears when closing the form
4. Observe being able to open the collapsed area with keyboard navigation

## Screenshots
<img width="639" alt="Screenshot 2024-09-25 at 12 58 50" src="https://github.com/user-attachments/assets/13dd2bc8-1dbf-4bdb-a9dc-8d6ad31d31db">

## Checklist
